### PR TITLE
First draft of an introduction post

### PR DESCRIPTION
**Pull request by 17cupsofcoffee**
I tried my best not to make it overlap too much with what @ozkriff is working on for the newsletter 😄 Let me know if the content seems right/if there's anything obvious I've missed.

I also tweaked a couple of style things for readability because they would have driven me crazy otherwise...

### CODE REVIEW

**Comment by icefoxen** [comment]
> We know it can be an intimidating and scary field when you're just starting out.

Suggest "large and intimidating" instead of "intimidating and scary". Game devs know no fear.

**Comment by 17cupsofcoffee** [comment]
I also like that phrasing better, I think! Happy to change it, but we should also update the actual charter/README to match - I just copied and pasted from there :)

File: _posts/2019-08-17-introducing-the-rust-game-development-working-group.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+[Our charter](https://github.com/rust-gamedev/wg/blob/master/charter.md) sets out two primary goals:
+
+* To improve the experience of people using Rust for game development. We want game developers to have a representation in the Rust project. We are a bridge between them and the Rust dev team.
+* To share knowledge and make game engine development more accessible for people that are not engine developers. We know it can be a large and intimidating field when you're just starting out.
+
+## What have we been up to?
+
+While the working group is still in its infancy, we've already got several projects underway:
+
+* Perhaps the biggest success so far has been the creation of [`raw-window-handle`](https://github.com/rust-windowing/raw-window-handle), a library that provides a common interface that windowing libraries (such as `winit`) can use to talk to graphics libraries (such as `gfx-hal`). [Work is currently ongoing](https://github.com/rust-gamedev/wg/issues/26) to integrate this into various libraries, and once this is complete, it should be a big step towards making the ecosystem more decoupled.
```
Comment (AlexEne) [inline _posts/2019-08-17-introducing-the-rust-game-development-working-group.md]: I think we can remove the perhaps :)

Comment (17cupsofcoffee) [reply _posts/2019-08-17-introducing-the-rust-game-development-working-group.md]: Fixed :)

**Pull request merged**